flash/vision/classification/data.py

```diff
@@ -58,6 +58,19 @@ def __init__(
         predict_transform: Optional[Union[Dict[str, Callable]]] = None,
         image_size: Tuple[int, int] = (196, 196),
     ):
+        """
+        Preprocess pipeline definition for image classification tasks.
+
+        Args:
+            train_transform: Dictionary with the set of transform to apply during training. Default: None.
+            val_transform: Dictionary with the set of transform to apply during validation. Default: None.
+            test_transform: Dictionary with the set of transform to apply during testing. Default: None.
+            predict_transform: Dictionary with the set of transform to apply during prediction. Default: None.
+            image_size: A tuple with the expected output image size. Default: (196, 196).
+
+        Returns:
```

Comment:
Don't need a returns for `__init__`, [PEP](https://www.python.org/dev/peps/pep-0484/#the-meaning-of-annotations) says typing should be `-> None`